README.md

```diff
@@ -50,101 +50,110 @@ CC=`which gcc` CXX=`which g++` cmake ..
 make
 ```
 
-## Testing
-### Running Tests
-to run tests:
-```bash
-make test
-# or
-ninja test
-```
-or using the ctest environment:
+AutoPas relies on a small number of dependencies. By default AutoPas looks for
+installed versions of those libraries but it can also be forces to (selecively)
+use bundled versions. To make use of this feature call `cmake` with:
```

Comment:
```suggestion
installed versions of those libraries but it can also be forced to (selectively)
use bundled versions. To make use of this feature, call `cmake` with:
```